### ISSUE TITLE
set up a custom domain

### ISSUE THREAD

**Issue by afeld** [18F/linkify-citations#6]
If this is going to be used broadly or long-term, we probably don't want it to be tied to S3 (from the user's perspective). We might want to set up a custom domain, in case we would need to change where it's hosted later.

**Comment by vdavez** [comment]
Good point... can we point to an 18f.gov subdomain? open to suggestions about where it should live.

**Comment by afeld** [comment]
Actually, with https://github.com/18F/pages/pull/36 in, might be worth considering publishing with [18F Pages](https://github.com/18F/pages#adding-a-new-site).

/cc @mbland

**Comment by mbland** [comment]
Yep, and 18F/pages#36 is now running live in production, waiting for its first client.

**Comment by adelevie** [comment]
+1 to 18F Pages.

**Comment by vdavez** [comment]
Where did we end up on this? pages? s3?

**Comment by mbland** [comment]
It's live on Pages: https://pages.18f.gov/linkify-citations/linkify.min.js

If you want to go the custom domain route one day...let's talk. I don't think it'd be that hard at all to get Pages to support that.

**Comment by vdavez** [comment]
So @adelevie, how do you want to do this? We can use the `18f-pages` branch and push to S3 on merges to the branch?

**Comment by mbland** [comment]
Just to be clear: Pages doesn't use S3, but merges to `18f-pages` will automatically appear.

**Comment by vdavez** [comment]
Right, but Travis is configured to push to S3 on success.
